modules/javafx.graphics/src/main/java/javafx/css/Rule.java

```diff
@@ -361,6 +379,15 @@ final void writeBinary(DataOutputStream os, StyleConverter.StringStore stringSto
     static Rule readBinary(int bssVersion, DataInputStream is, String[] strings)
             throws IOException
     {
+        MediaRule mediaRule = null;
+
+        if (bssVersion >= 7) {
```

Comment:
please add a comment pointing to Stylesheet.BINARY_CSS_VERSION.

Ideally, there would be constants declared somewhere with explanation instead of the magic number '7'

Reply:
I've added comments to that effect.